### PR TITLE
fix(FEV-1653): when switch from Live entry to VOD then back to Live, all plugins icons are missing

### PR DESCRIPTION
**Pull request by semarche-kaltura**
### Description of the Changes

When player preset changes (f.e. from Playback to Live, etc) it makes UI re-initialise components. It causes next issue: if any component keep data in state - after player preset got changed the UI component loses all data that state has.
The goal of fix: instead of keeping data inside state of UI component take data from plugin class.

Solves FEV-1653
### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/services/upper-bar-manager/ui/displayed-bar/displayed-bar.component.tsx [hunk 1]
```diff
@@ -85,7 +78,7 @@ export class DisplayedBar extends Component<DisplayedBarProps, DisplayedBarState
             </IconWrapper>
           );
         })}
-        {dropdownControls.length && <MoreIcon icons={dropdownControls} />}
+        {dropdownControls.length > 0 && <MoreIcon icons={dropdownControls} />}
```
Comment (semarche-kaltura) [inline src/services/upper-bar-manager/ui/displayed-bar/displayed-bar.component.tsx:81]: fix for "0" that renders in DOM
Screenshot 2023-01-25 at 15 53 24

**Comment by SivanA-Kaltura** [comment]
@semarche-kaltura does dist need to be commited ?

**Comment by SivanA-Kaltura** [comment]
@semarche-kaltura so basically its the same code only moved around ?

**Comment by semarche-kaltura** [comment]
> @semarche-kaltura so basically its the same code only moved around ? was there any change in the logic itself ?

sure, take a look on description of the PR

**Comment by semarche-kaltura** [comment]
> @semarche-kaltura does dist need to be commited ?

oh, I see that Travis makes commits like "chore: update dist".
In any case it should not break anything

**Review by SivanA-Kaltura — APPROVED** [review]

File: src/services/upper-bar-manager/upper-bar-manager.tsx [hunk 2]
```diff
@@ -62,13 +61,18 @@ export class UpperBarManager {
     }
   }
 
-  private injectDisplayedBarComponentWrapper(iconsOrder: { [key in KalturaPluginNames | string]: number }): void {
+  private _getControls = (iconsOrder: IconsOrder) => {
```
Comment (JonathanTGold) [inline src/services/upper-bar-manager/upper-bar-manager.tsx:64]: Underscore is unnecessary here in TS

**Review by JonathanTGold — APPROVED** [review]

**Pull request merged**